app/components/collections/work_row_component.html.erb

```diff
@@ -2,9 +2,10 @@
   <%= render Works::LinkToShowComponent.new(work_version: work_version) %>
   <td>
     <% if allowed_to? :edit?, work_version %>
-      <%= link_to [:edit, work], aria: { label: "Edit #{Works::DetailComponent.new(work_version: work_version).title}" } do %>
-        <span class="fas fa-pencil-alt"></span>
-      <% end %>
+      <%= helpers.turbo_frame_tag dom_id(work, :edit),
```

Comment:
Now display work_type modal when clicking on the Edit button, fixes similar problem as described in ticket #1319

Reply:
I don't understand why this block needs to change.  The point of edit_button_work_path is to check the authorization of the current user, but here we're already doing that on line 4.  So, this is adding unnecessary complexity.

Reply:
Removed this from the current PR, my plan was to use the new `PurlCheckLinkComponent` instead but was running into a problem where Stimulus controller was properly injecting the modal action attribute into the modal. This can be addressed in a follow-up PR.